content/intro-to-storybook/react/ja/conclusion.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'まとめ'
+description: '今までの知識をまとめて、Storybook のテクニックをもっと学びましょう'
+---
+
+Storybook で最初の UI を作成しましたね。お疲れ様でした！ここまでの章で UI コンポーネントを作成し、複合させ、テストし、デプロイする方法を学びました。同じように進めていれば、リポジトリーとデプロイされた Storybook は以下のリンクと同じようになっていることでしょう。
+
+[📕 **GitHub のリポジトリー: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
+<br/>
+[🌎 **デプロイされた Storybook**](https://clever-banach-415c03.netlify.com/)
+
+Storybook は React、React Native、Vue、Angular、Svelte、その他のフレームワークにとって強力なツールです。開発者コミュニティーも活発でアドオンも充実しています。このチュートリアルで紹介した内容は、Storybook で出来ることの一部にすぎません。一度 Storybook を導入すれば、強固な UI を効率的に作れることに驚くことでしょう。
+
+## さらに学ぶには
+
+もっと深く掘り下げたいですか？役に立つリソースを紹介します。
+
+- [**Storybook の公式ドキュメント**](https://storybook.js.org/basics/introduction/)には API ドキュメント、コミュニティのリンク、アドオンのギャラリーがあります。
+
+- [**楽しい Storybook のワークフロー**](https://blog.hichroma.com/the-delightful-storybook-workflow-b322b76fd07)では Squarespace や、メジャーリーグサッカー、ディスカバリーネットワーク、 Apollo GraphQL といった効率の良いチームにおけるワークフローのベストプラクティスを紹介しています。
+
+- [**視覚的なテストのハンドブック**](https://www.chromatic.com/book/visual-testing-handbook)では、コンポーネントを Storybook で視覚的にテストする方法を掘り下げています。無料の 31 ページある eBook です。
```

Comment:
@chameleonhead if you don't mind, please adjust the link to point at https://www.learnstorybook.com/visual-testing-handbook/ instead of https://www.chromatic.com/book/visual-testing-handbook . To prevent a 404.